src/nni_manager/training_service/pai/paiTrainingService.ts

```diff
@@ -485,25 +238,11 @@ class PAITrainingService implements TrainingService {
                 }
             }
             await this.paiJobCollector.retrieveTrialStatus(this.paiToken, this.paiClusterConfig);
-            const restServer: PAIJobRestServer = component.get(PAIJobRestServer);
-            if (restServer.getErrorMessage !== undefined) {
-                throw new Error(restServer.getErrorMessage);
+            if (this.paiJobRestServer === undefined) {
+                throw new Error('paiBaseJobRestServer not implemented!');
```

Comment:
error message is not accurate,  should be paiJobRestServer  is undefined.

Reply:
fixed.